### PR TITLE
docker-compose scale is deprecated

### PR DESCRIPTION
**Pull request by steinbrueckri**
This command is deprecated. Use the up command with the --scale flag instead. Beware that using up with the --scale flag has some subtle differences with the scale command, as it incorporates the behaviour of the up command.

Ref.: https://docs.docker.com/compose/reference/scale/

### CODE REVIEW

**Review by onurctirtir — COMMENTED** [review]
LGTM, I'm ok with merging if looks good to @gurkanindibay as well.

**Comment by artyl** [comment]
You use the command from the example.
For a long time you can't understand why it doesn't work.
Finally you understand why and go to write a pull request.
In pull requests, you find a pull request that has been on hold for six months.
Thanks for the ruined time

**Review by gurkanindibay — APPROVED** [review]

**Comment by gurkanindibay** [comment]
@artyl  Thanks for your interest in the repo and I'm sorry for your lost time. 
I merged the PR
It was on me to check the correctness of the PR and I overlooked the issue. 
We have tasks needs to work on in daily basis and sometimes we can skip such PR's
Thanks for your reminder BTW

**Pull request merged**
